filebeat/module/elasticsearch/audit/ingest/pipeline-json.yml

```diff
@@ -3,6 +3,11 @@ processors:
   - json:
       field: message
       target_field: elasticsearch.audit
+  - dot_expander:
+      field: event.type
+      path: elasticsearch.audit
+  - drop:
+        if: '!["rest", "transport", "ip_filter", "security_config_change"].contains(ctx.elasticsearch?.audit?.event?.type)'
```

Comment:
it would be much easier to use `ctx.elasticsearch.audit.type != 'audit'`, but as I see this field was introduced only in version 7.3 - https://github.com/elastic/elasticsearch/pull/42887 (https://github.com/elastic/elasticsearch/blob/7.3/distribution/docker/docker-build-context/src/docker/config/log4j2.properties and in core - https://github.com/elastic/elasticsearch/blob/7.3/x-pack/plugin/core/src/main/config/log4j2.properties#L6), in version 7.1 it was not yet available https://github.com/elastic/elasticsearch/blob/7.1/x-pack/plugin/core/src/main/config/log4j2.properties

would it maybe be file to drop support for < 7.3 ?